src/main/scala/ScalaModulePlugin.scala

```diff
@@ -176,8 +174,12 @@ object ScalaModulePlugin extends AutoPlugin {
   private val runMimaIfEnabled = taskKey[Unit]("Run MiMa if mimaPreviousVersion and the module can be resolved against the current scalaBinaryVersion.")
 
   private lazy val mimaSettings: Seq[Setting[_]] = MimaPlugin.mimaDefaultSettings ++ Seq(
-    // manual cross-versioning because https://github.com/typesafehub/migration-manager/issues/62
-    mimaPreviousArtifacts := Set(organization.value % s"${name.value}_${scalaBinaryVersion.value}" % mimaPreviousVersion.value.getOrElse("dummy")),
+    mimaPreviousVersion := None,
+
+    {
+      import org.scalajs.sbtplugin.ScalaJSPlugin.autoImport._
+      mimaPreviousArtifacts := Set(organization.value %%% s"${moduleName.value}" % mimaPreviousVersion.value.getOrElse("dummy"))
```

Comment:
I think you should be able to use the following, which won't require the dependency on `sbt-scalajs`:
```scala
mimaPreviousArtifacts := Set(organization.value % moduleName.value % mimaPreviousVersion.value.getOrElse("dummy") cross crossVersion.value)
```